react/features/invite/functions.js

```diff
@@ -533,7 +534,7 @@ export function getShareInfoText(
 
             numbersPromise = Promise.all([
                 getDialInNumbers(dialInNumbersUrl, room, mucURL),
-                getDialInConferenceID(dialInConfCodeUrl, room, mucURL)
+                getDialInConferenceID(dialInConfCodeUrl, room, mucURL, locationURL.href)
```

Comment:
We'll be passing the whole URL with the params and everything this way. Is this OK? Do we need the whole URL?